site/app/templates/admin/admin_gradeable/AdminGradeableBase.twig

```diff
@@ -27,8 +27,8 @@
             <a class="nav-bar key_to_click" id="page_1_nav" onclick="onChangeNavTab(1);">Submissions/Autograding</a>
             <a class="nav-bar key_to_click" id="page_2_nav" onclick="onChangeNavTab(2);">Rubric</a>
             <a class="nav-bar key_to_click" id="page_3_nav" onclick="onChangeNavTab(3);">Grader Assignment</a>
-            <a class="nav-bar key_to_click" id="page_4_nav" onclick="onChangeNavTab(4);">Dates</a>
-            <a class="nav-bar key_to_click normal-btn{{(gradeable.isPeerGrading()) ? '' : ' hidden_no_peers'}}" id="page_5_nav" onclick="onChangeNavTab(5);">Peer Matrix</a>
+            <a class="nav-bar key_to_click normal-btn{{(gradeable.isPeerGrading()) ? '' : ' hidden_no_peers'}}" id="page_4_nav" onclick="onChangeNavTab(4);">Peer Matrix</a>
+            <a class="nav-bar key_to_click" id="page_5_nav" onclick="onChangeNavTab(5);">Dates</a>
```

Comment:
CSS class/Id name convention

Reply:
I changed the class name, but not the ID since it wouldn't match the previously named ID's existing in the file.